lib/EntryManager/EntrySelector.js

```diff
@@ -318,6 +318,7 @@ class EntrySelector extends React.Component {
       nameKey,
       addMenu,
       paneSetWidth
+      entryComponent
```

Comment:
```suggestion
      entryComponent: EntryComponent
```

In JSX, a lower-case tag is treated literally, so we need to rename this to start with a capital when destructuring.